packages/builders/scripts/testing/restart-stakeAtom.js

```diff
@@ -19,11 +19,11 @@ const trace = makeTracer('RestartSA', true);
  */
 export const restartStakeAtom = async ({
   consume: {
+    agoricNames,
     board,
     chainStorage,
     chainTimerService,
     cosmosInterchainService,
-
```

Comment:
fwiw the space was because `contractKits` is needed for restarting generally and not this particular restart